connect/runtime/src/main/java/org/apache/kafka/connect/runtime/TransformationChain.java

```diff
@@ -68,4 +74,11 @@ public int hashCode() {
         return Objects.hash(transformations);
     }
 
+    public String toString() {
+        StringJoiner str = new StringJoiner(", ", "<" + getClass().getSimpleName() + ": ", ">");
```

Comment:
I've found that avoiding using anything related to the type of a variable in its name is usually for the best. It takes a few more seconds to come up with an appropriate name, but it definitely pays off when reading, and even in generic uses as this one here it's still better IMO. E.g. I'd call this at least `chain` or something.

Reply:
I don't understand your comment -- can you explain what you mean by "avoiding using anything related to the type of a variable"?

Reply:
Oh, you mean the variable name "str"?